src/Parsers/ASTCreateUserQuery.cpp

```diff
@@ -55,6 +55,11 @@ namespace
                     password = authentication.getPasswordHashHex();
                     break;
                 }
+                case Authentication::LDAP_SERVER:
+                {
+                    password = authentication.getServerName();
```

Comment:
That's not a password in this case.
Maybe it's better to rename the variable `password` into something like `with_param`.

Reply:
Done. Renamed to `by_value`.